### PR TITLE
Clean up contraction usage, punctuations, etc. in `Silence`

### PR DESCRIPTION
**Pull request by agatemosu**
- EN: add etc. to "edit profile details", typo (e.g, -> e.g.), and remove . before parenthesis
- DE: etc. -> usw.
- ES: review
- ID: change to abbreviation (dan lain sebagainya/dan lain-lain -> dll.)

I don't know a better title

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Review by DoryanR — APPROVED** [review]
OK for the French file

**Comment by Nivalyx** [comment]
@agatemosu "dan lain-lain" in indonesian is `dll.` not `ddl.`

**Comment by agatemosu** [comment]
> @agatemosu "dan lain-lain" in indonesian is `dll.` not `ddl.`

sorry flr my typo xd

**Review by CloneWith — APPROVED** [review]
Chinese translation seems ok 👍

File: wiki/Silence/de.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Nutzer, die von Moderatoren stumm geschaltet werden, können folgende Dinge nich
 - In irgendeinem der osu!-Foren beitragen
 - Kommentare auf der Webseite hinterlassen (Neuigkeiten, Änderungsprotokoll, Beatmaps)
 - An [Beatmap-Diskussionen](/wiki/Beatmap_discussion) teilnehmen
-- Profildetails bearbeiten (z. B. der Avatar, die Sektion "ich!")
+- Profildetails bearbeiten (z. B. der Avatar, die Sektion "ich!", usw.)
```
Comment (The-Last-Cookie) [inline wiki/Silence/de.md:17]:
```suggestion
- Profildetails bearbeiten (z. B. der Avatar, die Sektion "ich!" usw.)
```

File: wiki/Silence/de.md [hunk 2]
```diff
@@ -34,8 +34,8 @@ Wenn du glaubst, mit Unrecht stumm geschaltet worden zu sein, dann kontaktiere d
 
 *Vorsicht: Unangemessenes Verhalten in privaten Nachrichten zu zeigen, ist nicht gegen die Regeln, solange dieses Verhalten nicht bei Leuten verwendet wird, die dies nicht möchten. Sofern dies gemeldet wird, werden entsprechende Strafen angewandt.*
 
-- Unangemessenes Verhalten in öffentlichen Chats oder Foren (z. B. in `#multiplayer`, Beatmap-Diskussionen, Kommentaren, usw.), einschließlich — aber nicht beschränkt auf — Spamming, Unsinn im Chat, Rassismus, etc.
-- Unangemessene Inhalte über andere Wege hochladen oder bereitstellen. (Z. B. Profilavatare, Einreichungen von Beatmaps, Profildetails, Banner, etc.).
+- Unangemessenes Verhalten in öffentlichen Chats oder Foren (z. B. in `#multiplayer`, Beatmap-Diskussionen, Kommentaren, usw.), einschließlich — aber nicht beschränkt auf — Spamming, Unsinn im Chat, Rassismus, usw.
+- Unangemessene Inhalte über andere Wege hochladen oder bereitstellen (z. B. Profilavatare, Einreichungen von Beatmaps, Profildetails, Banner, usw.).
```
Comment (The-Last-Cookie) [inline wiki/Silence/de.md:38]:
```suggestion
- Unangemessene Inhalte über andere Wege hochladen oder bereitstellen (z. B. Profilavatare, Einreichungen von Beatmaps, Profildetails, Banner usw.).
```

**Review by The-Last-Cookie — CHANGES_REQUESTED** [review]
Both `etc.` and `usw.` are fine for German, though there are no commas placed before them in enumerations (see ["Zeichensetzung am Ende von Aufzählungen" on duden.de](https://www.duden.de/sprachwissen/sprachratgeber/Zeichensetzung-am-Ende-von-Aufzahlungen)).

**Review by Walavouchey — APPROVED** [review]

**Comment by agatemosu** [comment]
thanks for the title

**Pull request merged**